testregistry.textproto

```diff
@@ -177,6 +177,18 @@ test: {
   readme: ""
   exec: " "
 }
+test: {
+  id: "BMP-2.7"
+  description: "BMP pre-policy test"
+  readme: ""
+  exec: " "
+}
+test: {
+  id: "BMP-2.8"
+  description: "BMP post-policy test"
+  readme: ""
+  exec: " "
+}
```

Comment:
Id's should match the READMEs;  (BMP-1.2, BMP-1.3)

Also fill in the `readme:` URL